File: net472/OwinParadise/OwinParadise/SignalR.cs
using Microsoft.Owin.Hosting;
using Owin;
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PortingParadise
{
    class SignalR
    {
        public class Startup
        {
            public void Configuration(IAppBuilder app)
            {
                app.MapSignalR();
            }
        }

        static void Main(string[] args)
        {
            string uri = "http://localhost:9999/";
            //using (WebApp.Start<Startup>(uri))
            //{
                Console.WriteLine("Started");
                // Open the SignalR negotiation page to make sure things are working.
                Process.Start(uri + "signalr/negotiate");
                Console.ReadKey();
                Console.WriteLine("Finished");
            //}
        }
    }
}

File: net472/OwinParadiseVb/OwinParadiseVb/SignalR.vb
Imports Microsoft.Owin.Hosting
Imports Owin

Namespace PortingParadise
    Public Class SignalR
        public class Startup
            public Sub Configuration(app As IAppBuilder)
                app.MapSignalR()
            End Sub

            public Sub Main(args  As String())
                Dim uri As String = "http://localhost:9999/"
                Using (WebApp.Start(Of Startup)(uri))
                    Console.WriteLine("Started")
                    Process.Start(uri + "signalr/negotiate")
                    Console.ReadKey()
                End Using
            End Sub
        End Class
    End Class
End Namespace